randovania/games/samus_returns/logic_database/Area 4 - West.txt

```diff
@@ -271,10 +271,13 @@ Extra - asset_id: collision_camera_006
   * Extra - actor_type: weightactivatedplatform
   * Extra - start_point_actor_name: ST_ChozoUnlockAreaDNA_Checkpoint
   > Chozo Seal Right
-      Space Jump or Can Spiderspark or Use Spider Ball
+      Any of the following:
+          Space Jump or Use Spider Ball
```

Comment:
Using Space Jump isn't difficult but it requires some relatively decent timing to avoid damage. Worth adding a movement beginner to that.

Reply:
Updated